app/src/pages/settings/index.tsx

```diff
@@ -0,0 +1,142 @@
+import {
```

Comment:
I think these should live at `/projects/[id]/settings`, or maybe `/projects/settings`? We'll have user (and actual org) settings as well in the future.

Reply:
Makes sense!

Reply:
It's at `/project/settings` now